examples/adaptive_library/adaptive_scripts/TrapezoidalDuct.py

```diff
@@ -0,0 +1,184 @@
+"""
+Trapezoidal ducts
+==============================================
+Adaptive Templates script to alter rectangular ducts to trapezoidal ducts.
```

Comment:
```suggestion
Trapezoidal ducts
============
Adaptive Templates script to alter rectangular ducts to trapezoidal ducts.
```
This example must have a description in a clear, simple but complete sentence. Something like: "This example shows how to use use adaptive template scripts to change rectangular ducts to trapezoidal ducts."